LBHFSSPublicAPI/V1/Gateways/ServicesGateway.cs

```diff
@@ -41,7 +41,7 @@ public SearchServiceGatewayResult SearchServices(SearchServicesRequest requestPa
                 .Include(s => s.ServiceLocations)
                 .Include(s => s.ServiceTaxonomies)
                 .ThenInclude(st => st.Taxonomy)
-                .Where(s => s.Status == "active")
+                .Where(s => s.Organization.Status.ToLower() == "active")
```

Comment:
Cool - this covers one of the ClickUp tasks - to make sure that paused orgs are not returned b y search.